shared/js/l10n.js

```diff
@@ -1226,8 +1226,8 @@
     var createEntry = this.isPseudo() ?
       createPseudoEntry.bind(this) : Resolver.createEntry;
 
-    for (var i = 0, node; node = ast[i]; i++) {
-      this.entries[node.$i] = createEntry(node, this.entries);
+    for (var i = 0; i < ast.length; i++) {
+      this.entries[ast[i].$i] = createEntry(ast[i], this.entries);
```

Comment:
If you're changing it here I think you should also change it in `addAST` in build/l10n/l10n.js.  However, note that this change isn't actually needed now since you've removed the reason why `node` could be falsy.